### PR TITLE
usnic: small fixes

### PR DESCRIPTION
**Pull request by goodell**
@jsquyres please review

### CODE REVIEW

File: prov/usnic/src/usdf_fabric.c [hunk 1]
```diff
@@ -839,17 +853,25 @@ usdf_fabric_open(struct fi_fabric_attr *fattrp, struct fid_fabric **fabric,
 	dp = __usdf_devinfo;
 	for (d = 0; d < dp->uu_num_devs; ++d) {
 		dep = &dp->uu_info[d];
+		if (dep->ue_dev)
+			USDF_DEBUG("XXX DJG dep->ue_dattr.uda_devname=%s\n",
+				dep->ue_dattr.uda_devname);
+		else
+			USDF_DEBUG("XXX DJG dep->ue_dev=%p\n", dep->ue_dev);
```
Comment (jsquyres) [inline prov/usnic/src/usdf_fabric.c]: Were these XXX DJG's left in by accident?  Seems somewhat weird that there's a case where dep->ue_dev would be NULL...?

File: prov/usnic/src/usdf_fabric.c [hunk 2]
```diff
@@ -913,6 +940,7 @@ usdf_fabric_open(struct fi_fabric_attr *fattrp, struct fid_fabric **fabric,
 	sin.sin_addr.s_addr = fp->fab_dev_attrs->uda_ipaddr_be;
 	fp->fab_arp_sockfd = socket(AF_INET, SOCK_DGRAM, 0);
 	if (fp->fab_arp_sockfd == -1) {
+		USDF_INFO("unable to initialize timer\n");
```
Comment (jsquyres) [inline prov/usnic/src/usdf_fabric.c]: Wrong string in this message -- should be about the failed socket()

**Comment by goodell** [comment]
Ouch, this is why we do reviews.  I'll fix those Monday.  As always, I should never hurry something on a Friday evening.

**Comment by goodell** [comment]
pushed an update, hopefully didn't miss anything this time

**Comment by jsquyres** [comment]
:+1:

**Pull request merged**